wrappers/rust/icicle-fields/icicle-babybear/Cargo.toml

```diff
@@ -8,7 +8,7 @@ homepage.workspace = true
 repository.workspace = true
 
 [dependencies]
-icicle-core = { workspace = true }
+icicle-core = { workspace = true, features = [] }
```

Comment:
Why is this necessary if there are no default feature on `core`?

Reply:
This is just a left over from when I did testing against plonky3. It actually compiled ark-ec even without the ark feature. Probably will remove that after the CI pass

Reply:
And ark-ec doesn't compile with plonky3